### ISSUE TITLE
A11y_VS Code: Copilot_Suggestion_ScreenReader : Screen reader is not announcing tooltip information when we press CTRL+K+I.

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#27353]
-A11yTCS; -A11ySev2; -DesktopApp; -ScreenReader; -WCAG1.3.1; -Win11; -Win32; -NVDA; -Info and Relationships; -A11yWCAG2.1; -SH-VS Code Copilot-Win32-July23; -VS Code 

--- Actual Result:​
Screen reader is not announcing tooltip information when we press CTRL+K+I.

--- Expected Result:​
Screen reader should announce the tooltip information when we press CTRL+K+I.

--- Environment Details:
Application Name: VS Code: Copilot
Visual studio code Insiders Version: 1.81.0(User Setup)
Microsoft Windows 11 Enterprise 22H2 Build 22621.1992
Screen reader: NVDA
Jaws

--- Repro Steps:
1.Open the "Visual Studio Code" dialog box.
2.In Visual Studio Code, create a new JavaScript (*.js) file.
3.In the code Paste this code function calculateDaysBetweenDates(begin, end) { and press enter.
4. A suggestion will appear.
5.Prees "Ctrl + K, Ctrl + I" and a tooltip will appear.
6.Verify whether the screen reader is announcing tooltip information or not.

--- User Impact:
Screen reader users will not get information available on tooltip.
